Upstream: Yes
Source: Gentoo
Reason: automagic acl dep

http://bugs.gentoo.org/show_bug.cgi?id=188993
http://cups.org/str.php?L2723+P0+S0+C0+I0+E0+M10+Qnumber%3A2723
will be fixed in 1.4*
original patch by Timo Gurr

diff -Naur cups-1.3.3/config-scripts/cups-common.m4 cups-1.3.3.new/config-scripts/cups-common.m4
--- cups-1.3.3/config-scripts/cups-common.m4	2007-09-28 22:40:03.000000000 +0200
+++ cups-1.3.3.new/config-scripts/cups-common.m4	2007-10-04 14:40:49.414154416 +0200
@@ -270,7 +270,12 @@
 dnl See if we have POSIX ACL support...
 SAVELIBS="$LIBS"
 LIBS=""
-AC_SEARCH_LIBS(acl_init, acl, AC_DEFINE(HAVE_ACL_INIT))
+
+AC_ARG_ENABLE(acl, [  --enable-acl            enable POSIX ACL support, default=auto])
+if test "x$enable_acl" != xno; then
+	AC_SEARCH_LIBS(acl_init, acl, AC_DEFINE(HAVE_ACL_INIT))
+fi
+
 CUPSDLIBS="$CUPSDLIBS $LIBS"
 LIBS="$SAVELIBS"
 

